data/helpers.d/network

```diff
@@ -0,0 +1,9 @@
+# Check availability of a web path
+#
+# example: ynh_webpath_available $domain$path
+#
+# usage: ynh_webpath_available $domain_and_path
```

Comment:
Following [this remark from Bram](https://github.com/YunoHost/yunohost/pull/267#discussion_r104339016), I think we should avoid merging the domain and path, to anticipate the upcoming urlavailable helper which take domain and path as separate arguments (make stuff a lot easier).

So we could simply have `ynh_webpath_available $domain $path` instead.